utils/build/docker/java/spring-boot/src/main/java/com/datadoghq/system_tests/springboot/AppSecIast.java

```diff
@@ -299,6 +300,19 @@ void secureXSS(final ServletResponse response) throws IOException {
         xssExamples.secureXSS(response.getWriter());
     }
 
+    @GetMapping(value = "/hstsmissing/test_insecure", produces = "text/html")
+    public String hstsHeaderMissingInsecure(HttpServletResponse response) {
+      response.setStatus(HttpStatus.OK.value());
+      return "ok";
+    }
+
+    @GetMapping(value = "/hstsmissing/test_secure", produces = "image/gif")
```

Comment:
```suggestion
    @GetMapping(value = "/hstsmissing/test_secure", produces = "text/html")
```

Set it up so that we really verify the header is checked.

Reply:
I implemented the header in the endpoint but missed this change. It is fixed now.